core/server/models/base/index.js

```diff
@@ -374,22 +374,27 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
         return attrs;
     },
 
-    // Sets given values to `null`
-    setEmptyValuesToNull: function setEmptyValuesToNull() {
-        var self = this,
-            attr;
-
-        if (!this.emptyStringProperties) {
-            return;
-        }
-
-        attr = this.emptyStringProperties();
-
-        _.each(attr, function (value) {
-            if (self.get(value) === '') {
-                self.set(value, null);
-            }
+    // Gets nullable strings
+    getNullableStringProperties: _.memoize(function getNullableStringProperties(options) {
+        return options.query.columnInfo().then((columns) => {
```

Comment:
Using options.query here means that if the query is part of a transaction, this will be part of it too.

Reply:
Why do we have to query the database to get the "nullable" information? 🤔 
Can't we simply ask the schema?

Reply:
Yeah IMO model layer is allowed to access the schema.